fe/fe-core/src/main/java/org/apache/doris/statistics/BaseAnalysisTask.java

```diff
@@ -222,23 +226,23 @@ protected String getDataSizeFunction(Column column) {
         return "COUNT(1) * " + column.getType().getSlotSize();
     }
 
-    protected String getSampleExpression() {
+    protected TableSample getTableSample() {
         if (info.forceFull) {
-            return "";
+            return null;
         }
-        int sampleRows = info.sampleRows;
+        long sampleRows = info.sampleRows;
         if (info.analysisMethod == AnalysisMethod.FULL) {
             if (Config.enable_auto_sample
                     && tbl.getDataSize(true) > Config.huge_table_lower_bound_size_in_bytes) {
                 sampleRows = Config.huge_table_default_sample_rows;
             } else {
-                return "";
+                return null;
             }
         }
         if (info.samplePercent > 0) {
-            return String.format("TABLESAMPLE(%d PERCENT)", info.samplePercent);
+            return new TableSample(true, (long) info.samplePercent);
```

Comment:
1. samplePercent has higher priority than that of sampleRows. Do we have document about this?
2. It would be better if we could convert percentage to sampleRows here. for example, sample 5% means sampleRows = table_rows * 5%

Reply:
1. Sample percent doesn't have higher priority, we sample by rows by default, if user set sample by percent explicitly then sample by percent, docs has already explained this
2. sample rows by default has been fully discussed in the group